src/meta/meta_bulk_load_service.cpp

```diff
@@ -59,6 +59,7 @@ void bulk_load_service::on_start_bulk_load(start_bulk_load_rpc rpc)
     error_code e = check_bulk_load_request_params(request.app_name,
                                                   request.cluster_name,
                                                   request.file_provider_type,
+                                                  request.remote_root_path,
```

Comment:
Is there any compatiable issue when an old shell tools send a bulk load request?

Reply:
Yes, the old shell tools will send invalid request lack of remote_root_path, I will add related code for shell command. Consider that bulk load is not already used in production and we plan to use admin-cli tool instead of shell tools, and bulk load related command has not already been implemented in admin-cli tool, I think this compatiable problem can be acceptable.